extensions/s3/src/test/java/com/hazelcast/jet/s3/S3TestBase.java

```diff
@@ -168,6 +170,23 @@ void deleteBucket(S3Client client, String bucket) {
         }
     }
 
+    Stream<String> S3ObjectToLines(S3Object o, S3Client client, String bucketName) {
+        try {
+            ResponseInputStream<GetObjectResponse> is = client.getObject(req -> req.bucket(bucketName).key(o.key()), toInputStream());
+            try (BufferedReader reader = new BufferedReader(new InputStreamReader(is, CHARSET))) {
```

Comment:
why copy/paste `inputStreamToLines` here ? we can just call it 
`return inputStreamToLines(client.getObject(req -> req.bucket(bucketName).key(o.key()), toInputStream()));`